src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -41,25 +45,43 @@
     <DebugTestFrameworkFolder>net45</DebugTestFrameworkFolder>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(TestWithCore)' == 'true'">
-    <TestTargetFramework Include=".NETCoreApp,Version=v1.0">
+  <ItemGroup>
+    <TestNugetTargetMoniker Include=".NETCoreApp,Version=v1.0" Condition="'$(TestTFM)' == 'netcoreapp1.0'">
       <Folder>netcoreapp1.0</Folder>
-    </TestTargetFramework>
-  </ItemGroup>
-
-  <ItemGroup Condition="'$(TestWithCore)' != 'true'">
-    <TestTargetFramework Include=".NETFramework,Version=v4.6.2" Condition="'$(TargetGroup)' == 'net462'">
+    </TestNugetTargetMoniker>
+    <TestNugetTargetMoniker Include=".NETFramework,Version=v4.6.3" Condition="'$(TestTFM)' == 'net463'">
+      <Folder>net462</Folder>
+    </TestNugetTargetMoniker>
+    <TestNugetTargetMoniker Include=".NETFramework,Version=v4.6.2" Condition="'$(TestTFM)' == 'net462'">
       <Folder>net462</Folder>
-    </TestTargetFramework>
-    <TestTargetFramework Include=".NETFramework,Version=v4.6.1" Condition="'$(TargetGroup)' == 'net461'">
+    </TestNugetTargetMoniker>
+    <TestNugetTargetMoniker Include=".NETFramework,Version=v4.6.1" Condition="'$(TestTFM)' == 'net461'">
       <Folder>net461</Folder>
-    </TestTargetFramework>
-    <TestTargetFramework Include=".NETFramework,Version=v4.6" Condition="'$(TargetGroup)' == 'net46'">
+    </TestNugetTargetMoniker>
+    <TestNugetTargetMoniker Include=".NETFramework,Version=v4.6" Condition="'$(TestTFM)' == 'net46'">
       <Folder>net46</Folder>
-    </TestTargetFramework>
-    <TestTargetFramework Include=".NETFramework,Version=v4.5" Condition="'$(TargetGroup)' == 'net45'">
+    </TestNugetTargetMoniker>
+    <TestNugetTargetMoniker Include=".NETFramework,Version=v4.5" Condition="'$(TestTFM)' == 'net45'">
       <Folder>net45</Folder>
-    </TestTargetFramework>
+    </TestNugetTargetMoniker>
+    <TestNugetTargetMoniker Include=".NETFramework,Version=v4.5.1" Condition="'$(TestTFM)' == 'net451'">
+      <Folder>net451</Folder>
+    </TestNugetTargetMoniker>
+    <TestNugetTargetMoniker Include=".NETFramework,Version=v4.5.2" Condition="'$(TestTFM)' == 'net452'">
+      <Folder>net451</Folder>
+    </TestNugetTargetMoniker>
+    <TestNugetTargetMoniker Include="UAP,Version=v10.0" Condition="'$(TestTFM)' == 'netcore50aot'">
+      <Folder>netcore50aot</Folder>
+    </TestNugetTargetMoniker>
+    <TestNugetTargetMoniker Include=".NETStandard,Version=v1.3" Condition="'$(TestTFM)' == 'netstandard13aot'">
```

Comment:
This was during an initial changeset, I wasn't sure if we wanted to have TestTFMs that would map to non-corresponding monikers (like in the case of aot). The netstandardaot13 was based on the TargetGroup filtering done in dir.props. https://github.com/dotnet/corefx/blob/1591c855ceb5a6a1547ae2f1cb2ef55b037cc48c/dir.props#L415